lib/elastic_apm/config.rb

```diff
@@ -156,6 +156,16 @@ def collect_metrics?
       metrics_interval > 0
     end
 
+    def disabled_instrumentations
+      disable_instrumentations
+    end
+
+    def disabled_instrumentations=(value)
+      warn '[DEPRECATED] The option disabled_instrumentations has been ' \
+        'renamed to disable_instrumentations to align with other agents.'
+      self.disable_instrumentations = value
+    end
+
```

Comment:
Can't you just use the `Deprecations` module here?

Reply:
I could perhaps but I wanted a special warning message.